.github/workflows/NodeCI.yml

```diff
@@ -49,6 +49,18 @@ jobs:
                   npm install
             - name: Test
               run: npm test
+    test-with-eslint8:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+            - name: Install Target Packages
+              run: |+
+                  npm i -D eslint@^8.0.0-0
+                  npx rimraf node_modules
+                  npm install
```

Comment:
What effect does this have? (Same for ESLint v6.x.) FWIK, this should be a no-op.

Reply:
It was a workaround to make it work when package-lock.json does not exist. It certainly may not be necessary now😅

Reply:
I changed it and it seems to work fine 😄